### PR TITLE
Hierarchical planning notebook

### PR DESCRIPTION
**Pull request by arbimo**
Colab link (works as long as the `hierarchical-notebook` branch exists)
https://colab.research.google.com/github/aiplan4eu/unified-planning/blob/hierarchical-notebook/notebooks/Hierarchical_Planning.ipynb

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -37,6 +37,16 @@ In this notebook we show how to use the unified planning library to model tempor
 In particular we model the well-known MatchCellar problem and we call a planner to solve it.
 
 
+## Hierarchical Planning
+
+[![Open In GitHub](https://img.shields.io/badge/see-Github-579aca?logo=github)](Unified_Planning_Temporal.ipynb)
```
Comment (alvalentini) [inline notebooks/README.md:42]:
```suggestion
[![Open In GitHub](https://img.shields.io/badge/see-Github-579aca?logo=github)](Hierarchical_Planning.ipynb)
```

**Comment by arbimo** [comment]
@alvalentini You can use the notebook here for the hands-on session. A colab link is provided in the initial comment. I will update this branch to fix a few remaining inconsistencies in the notebook.

**Comment by alvalentini** [comment]
@arbimo Perfect, thanks!

**Comment by arbimo** [comment]
@alvalentini This is now ok to be merged I believe.

**Pull request merged**